File: wallet/src/main/java/com/worldventures/wallet/analytics/new_smartcard/UnAssignCardSuccessGetStartedAction.java
package com.worldventures.wallet.analytics.new_smartcard;

import com.worldventures.janet.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.Attribute;
import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.wallet.analytics.WalletAnalyticsAction;

@AnalyticsEvent(action = "wallet:settings:general:setup new smartcard:existing card detected:unassign successful:get started",
                trackers = AdobeTracker.TRACKER_KEY)
public class UnAssignCardSuccessGetStartedAction extends WalletAnalyticsAction {

   @Attribute("unassigncardstep7")
   final String unAssignCardStep7 = "1";
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/new_smartcard/UnAssignCardSuccessGetStartedAction.kt
package com.worldventures.wallet.analytics.new_smartcard

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.Attribute
import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.wallet.analytics.WalletAnalyticsAction

@AnalyticsEvent(action = "wallet:settings:general:setup new smartcard:existing card detected:unassign successful:get started",
      trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class UnAssignCardSuccessGetStartedAction : WalletAnalyticsAction() {

   @Attribute("unassigncardstep7") internal val unAssignCardStep7 = "1"
}
